src/AccountManager.cpp

```diff
@@ -109,7 +109,7 @@ void AccountManager::raiseManageAccountsDialog()
 
 void AccountManager::switchAccount(const Account & account)
 {
-    QNDEBUG(QStringLiteral("AccountManager::switchAccount: ") << account);
+    QNDEBUG(QStringLiteral("AccountManager::switchAccount: ") << account.name());
```

Comment:
I think in this particular place it would be more useful to print the entire account because here not only the name but also the type of the account we are switching to matters a lot. Maybe you could add printing of account type here.